src/rust/ide/src/model/module.rs

```diff
@@ -422,10 +436,10 @@ pub trait API:Debug {
 // === Setters ===
 
     /// Update whole content of the module.
-    fn update_whole(&self, content:Content);
+    fn update_whole(&self, content:Content) -> FallibleResult<()>;
```

Comment:
why FailibleResult is not parametrized with `()` by default?

Reply:
Because `std::Result` is not. And I like it: because seeing "FallibleResult" alone does not tells much what the result is. Looking at "FallibleResult<()>" people know at glimpse that function returns unit (or error).